src/basics/entry-points.md

```diff
@@ -44,25 +45,26 @@ impl CounterContract {
 }
 ```
 
-Note that `#[entry_points]` is added above the `#[contract]`. This is because `contract` removes
-the attributes such as `#[msg(...)]` on which both those macros rely. Remember to always place 
-`#[entry_points]` first.
+Note that **`#[entry_points]`** is added above the **`#[contract]`**.
+It is because **`#[contract]`** removes attributes like **`#[msg(...)]`** on which both these macros rely.
+
+Always remember to place **`#[entry_points]`** first.
 
-`Sylvia` generates the entry_points with [`#[entry_point]`](https://docs.rs/cosmwasm-std/1.3.1/cosmwasm_std/attr.entry_point.html)
-attribute macro. Its purpose is to wrap the whole entry point to the form Wasm runtime understands. 
+[Sylvia](https://github.com/CosmWasm/sylvia) generates entry points with [`#[entry_point]`](https://docs.rs/cosmwasm-std/1.3.1/cosmwasm_std/attr.entry_point.html)
+attribute macro. Its purpose is to wrap the whole entry point to the form the Wasm runtime understands. 
 The proper Wasm entry points can use only basic types supported natively by Wasm specification, and 
 Rust structures and enums are not in this set. Working with such entry points would be 
 overcomplicated, so CosmWasm creators delivered the `entry_point` macro. It creates the raw Wasm 
 entry point, calling the decorated function internally and doing all the magic required to build our 
 high-level Rust arguments from arguments passed by Wasm runtime.
 
-Now that our contract has a proper `entry point` let's build it and check if it's properly defined.
+Now, when our contract has a proper entry point, let's build it and check if it's correctly defined:
 
-```
-contract $ cargo build --release --target wasm32-unknown-unknown --lib
+```shell
+[contract]$ cargo build --release --target wasm32-unknown-unknown --lib
```

Comment:
please keep it in the same form. Every has different terminal setup and I would like to keep this minimal one `contract $`